lambdatest/manifest.json

```diff
@@ -27,7 +27,9 @@
   "is_public": true,
   "integration_id": "lambdatest",
   "assets": {
-    "dashboards": {},
+    "dashboards": {
+      "LambdaTest": "assets/dashboards/overview.json"
+    },
     "saved_views": {},
     "monitors": {},
     "service_checks": "assets/service_checks.json",
```

Comment:
With the logs pipeline now added, can you update the `logs` line below to say:

```
    "logs": {
        "source": "lambdatest"
    },
```

Reply:
@prateeksainilambdatest in case you didn't get a notification earlier, this update will help link the integration to the new logs pipeline.

Reply:
Updated